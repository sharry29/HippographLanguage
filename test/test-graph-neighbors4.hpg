int main() {
	graph<int> g = [1 -()- 2 -()- 3 -()- 1];
	graph<int> g_sub1 = g.neighbors(1; 3; false);
	for_node (n : g_sub1) {
		n.print();
		print("");
	}
	print("");
	graph<int> g_sub2 = g.neighbors(1; 3; true);
	for_node (n : g_sub2) {
		n.print();
		print("");
	}
}