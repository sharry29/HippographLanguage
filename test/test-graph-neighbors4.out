2:null
3:null

1:null
2:null
3:null
